### ISSUE TITLE
locked linenumbers on inner margin

### ISSUE THREAD

**Issue by floriandk** [maieul/ledmac#727]
To be able to work with the original ms's linenumbers, I use `\endlock`/`\startlock` together with `\advanceline` or `\setline`, which works fine:

```latex
\documentclass[15pt]{scrbook}

\usepackage[a6paper]{geometry}

\usepackage[series={A},noeledsec,nofamiliar,noledgroup]{reledmac}

\newcommand{\lbms}{\endlock\advanceline{1}\startlock / }

\firstlinenum{1}
\linenumincrement{1}
%\linenummargin{inner}


\begin{document}
\sloppy

\beginnumbering
\pstart\setline{1}\startlock
Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Morbi\lbms
commodo, ipsum sed pharetra gravida, orci magna rhoncus neque, id\lbms
pulvinar odio lorem non turpis. Nullam sit amet enim. Suspendisse id\lbms
velit vitae ligula volutpat condimentum. Aliquam erat volutpat. Sed\lbms
quis velit. Nulla facilisi. Nulla libero. Vivamus pharetra posuere\lbms
sapien. Nam consectetuer. Sed aliquam, nunc eget euismod ullamcorper,\lbms
lectus nunc ullamcorper orci, %
\edtext{fermentum bibendum enim nibh eget ipsum.\lbms
Donec porttitor ligula eu dolor. Maecenas vitae nulla consequat libero\lbms
cursus venenatis.}{%
	\lemma{fermentum \ldots{} venenatis.}
	\Afootnote{\emph{sic.}}}
 Nam magna enim, accumsan eu, blandit sed, blandit a,\lbms
eros.\lbms
\pend
\pstart
Quisque facilisis erat a dui. Nam malesuada ornare dolor. Cras\lbms
libero. Integer nec enim. Phasellus aliquam enim et tortor. Quisque\lbms
consequat rutrum. Nullam egestas feugiat felis. Integer adipiscing\lbms
gravida, diam sit amet rhoncus ornare, id\lbms
egestas pede nibh eget odio. %
\edtext{Proin tincidunt, velit vel porta\lbms
elementum, magna diam molestie sapien, non aliquet massa pede eu diam.\lbms
Aliquam iaculis. Fusce et a est quis orci\lbms
}{%
	\lemma{Proin \ldots{} orci}%
	\Afootnote[nosep]{= A}}
ipsum et nulla tristique facilisis. Donec\lbms
eget sem sit amet ligula viverra gravida. %
\edtext{Etiam vehicula urna vel\lbms
turpis. Suspendisse sagittis ante a urna. Morbi}{%
	\lemma{Etiam \ldots{} Morbi}%
	\Afootnote{= B}}
semper ligula. Nunc molestie, nisl sit amet cursus convallis, sapien\lbms
lectus pretium metus, vitae pretium enim wisi id lectus. Donec\lbms
vestibulum. Etiam vel nibh. Nulla facilisi. Mauris pharetra. Donec\lbms
augue. Fusce ultrices, neque id dignissim ultrices, tellus mauris\lbms
dictum elit, vel lacinia enim metus eu nunc.\lbms
\pend
\pstart
Proin at eros non eros adipiscing mollis. Donec semper turpis sed\lbms
diam. Sed consequat ligula nec tortor. Integer eget sem. Ut vitae enim\lbms
eu est vehicula gravida. Morbi ipsum ipsum, porta nec, tempor id,\lbms
auctor vitae, purus. Pellentesque neque. Nulla luctus erat vitae\lbms
aliquet, quam elementum condimentum feugiat, tellus odio consectetuer\lbms
wisi, vel nonummy sem neque in elit. Curabitur eleifend wisi iaculis\lbms
ipsum. Pellentesque habitant morbi tristique senectus et netus et\lbms
malesuada fames ac turpis egestas. In non velit non ligula laoreet\lbms
ultrices. Praesent ultricies facilisis nisl. Vivamus luctus elit sit\lbms
amet mi. Phasellus pellentesque, erat eget elementum volutpat, dolor\lbms
nisl porta neque, vitae sodales ipsum nibh in ligula. Maecenas mattis\lbms
pulvinar diam. Curabitur sed leo.\lbms
\endlock
\pend
\endnumbering

\end{document}
```

But when I use `\linenummargin{inner}` on the same example the linenumber 8, 16 and 17 end up in the wrong margin. This is always the case when the `\edtext` stretches over the pagebreak. `\edtext`s in the middle of the page (note for ll. 19--20) don't give problems.

Is this a bug or could you perhaps suggest another approach that works with the current `reledmac`? 

Like always: Many thanks!

**Comment by maieul** [comment]
First, I am very surprised you didnt 'use the pref-full form to open an other issue. Is

To deal with original line number, you could use the linenumberannotation mechanism, which will allow you to have both original line number and moderne line number. 

In any case, I think there is a bug. I look on.

**Comment by floriandk** [comment]
> First, I am very surprised you didnt 'use the pref-full form to open an other issue. Is

I'm afraid I'm not following… If there is a new form for issues, I just haven't noticed it yet.

> To deal with original line number, you could use the linenumberannotation mechanism, which will allow you to have both original line number and moderne line number.

Thanks for the hint, I'll look into it! (This was probably not available yet back when I started setting up this text.)

**Comment by maieul** [comment]
You should have this field when opening an issue
![screenshot-2017-11-24 maieul ledmac](https://user-images.githubusercontent.com/636640/33219513-d1fcb97a-d142-11e7-8b0d-1ec7036a7bc2.png)

The linenumberannotation mechanism was released 2 weeks ago.

**Comment by floriandk** [comment]
I have no idea why but I didn't see this form. When I _now_ press "new issue", it comes up, though. A glitch on GitHub?

Anyway , to complete:

# Version of reledmac / reledpar
2017/11/14 v2.26.0a but also earlier

# Does the problem happen recently? Precise in which version the problem didn't exist.
I only have two other versions readily availabe. v2.18.1 (TL 16) shows the same wrong behaviour. 
 v2.9.0 (TL15) shows a different but also wrong behaviour: ll. 1--7 correct, 8--15 wrong (outer margin), rest correct again.

# Does the problem happen only when combining with some packages?
no

# Does the problem happens both in parallel typesetting and normal typesetting?
no! With `reledpar` and `\linenummargin*{inner}` the linenumbers appear correctly on the inner margin.

**Comment by maieul** [comment]
thanks, I will try to get time for this problem soon (but i can't 
garantee you)

**Comment by floriandk** [comment]
PS:

I have now tried the `\linenumannotation`-mechanism -- very useful indeed for the usages mentioned in section 9.10 of the manual.

Unfortunately it isn't a solution for my case though, as it afaics only allows me to _add_ alternative numbering to the ones calculated by `reledmac`: 
To avoid confusion, I want to use the manuscript's linenumbers _only_, not the new ones from the edition.

If the above bug is difficult to fix, would it instead perhaps be easier to add another option `\Xlinenumannotationposition{replace}` that would completely cancel out automatic linenumbers from printing (in the margin and in critical notes) and completely replace them with the ones entered by the user via `\linenumannotation` ?

**Comment by maieul** [comment]
@floriandk I think we can close now, there is https://tex.stackexchange.com/questions/468970/bible-verse-numbers-in-margin-with-reledmac which show you an example to solve your problem (it is verse number, not line number, but the problem is the same)
